### PR TITLE
feat: add Pest options to help output

### PR DESCRIPTION
**Pull request by owenvoke**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Fixed tickets | #... 

This adds an additional help section to the output of `pest --help`.

Preview screenshot of help command output

### CODE REVIEW

File: tests/.snapshots/success.txt [hunk 1]
```diff
@@ -348,6 +348,9 @@
   ✓ it throws exception when `process isolation` is true
   ✓ it do not throws exception when `process isolation` is false
 
+   PASS  Tests\Unit\Console\Help
+  ✓ it outputs the help information when --help is used
```
Comment (nunomaduro) [inline tests/.snapshots/success.txt:352]: We may need an integration test on this. Do you mind of adding it?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: tests/.snapshots/success.txt (hunk 1 above)
Comment (owenvoke) [reply tests/.snapshots/success.txt:352]: Sure. :+1: So just call `pest --help` and check it's in the output?

Comment (owenvoke) [reply tests/.snapshots/success.txt:352]: Is that ok? Or should they be in the same file? 🤔

File: tests/Visual/Help.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+test('visual snapshot of help command output', function () {
+    $output = function () {
+        $process = (new Symfony\Component\Process\Process(['php', 'bin/pest', '--help', '--colors=never']));
+
+        $process->run();
+
+        return $process->getOutput();
+    };
+
+    expect($output())->toContain('Pest Options:');
```
Comment (nunomaduro) [inline tests/Visual/Help.php:12]: I would prefer this line to contain a snapshot. So we can see the entire thing. Sorry.

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: tests/Visual/Help.php (hunk 2 above)
Comment (owenvoke) [reply tests/Visual/Help.php]: No problem 👍🏻 Updated.

**Review by nunomaduro — APPROVED** [review]

**Pull request merged**